builders/integrations/indexers/covalent.md

```diff
@@ -1,30 +1,40 @@
-- [Increment](#increment) - create and embed custom charts with no-code analytics
+- [Unified API](#unified-api-overview) - incorporate blockchain data into your app with a familiar REST API
 
-This guide will cover all of the details needed to get started with the [Covalent API](https://www.covalenthq.com/docs/api){target=\_blank} and how to access the API endpoints for Moonbeam using curl commands and JavaScript and Python snippets.
+This guide will cover all of the details needed to get started with the [GoldRush API](https://goldrush.dev/docs/unified-api/){target=\_blank} and how to access the API endpoints for Moonbeam using curl commands and JavaScript and Python snippets.
 
 --8<-- 'text/_disclaimers/third-party-content-intro.md'
 
-## Unified API {: #unified-api }
+## Unified API Overview {: #unified-api-overview }
 
-Covalent's Unified API is a powerful but easy-to-use REST API that offers visibility to assets across all blockchain networks. It features a consistent request and response object format across networks. For example, a user can fetch all the token balances for a wallet address across any supported blockchain by changing the unique blockchain name or id path parameter in the request URL. Covalent's Unified API can offer more data flexibility than JSON-RPC interfaces, which are typically limited to queries on a specific block. It also allows queries on multiple objects and batch exports of data.
+GoldRush's Unified API is a powerful but easy-to-use REST API that offers visibility to assets across all blockchain networks. It features a consistent request and response object format across networks. For example, a user can fetch all the token balances for a wallet address across any supported blockchain by changing the unique blockchain name or ID path parameter in the request URL. GoldRush's Unified API can offer more data flexibility than JSON-RPC interfaces, which are typically limited to queries on a specific block. It also allows queries on multiple objects and batch exports of data.
 
-[![Example API response in JSON](/images/builders/integrations/indexers/covalent/covalent-1.webp)](https://www.covalenthq.com/docs/api/balances/get-token-balances-for-address/?utm_source=moonbeam&utm_medium=partner-docs){target=\_blank} *Click on the above image to try out the request yourself.*
+### Querying the Unified API {: #querying-the-unified-api }
+
+It's easy to get started querying the Unified API after you've secured a Goldrush API Key. Make sure you have [your API Key](https://goldrush.dev/platform/auth/register/){target=\_blank} which begins with `cqt_` or `ckey_`. 
+
+You can interact with any of the API methods in the web interface of the Goldrush docs. To try out the token balances API, head to the [Token Balances docs](https://goldrush.mintlify.app/docs/api/balances/get-token-balances-for-address){target=\_blank} and take the following steps:
```

Comment:
Token Balances only needs to be capitalized if it's the title of the page you're point to. It looks like it's different so you don't need to capitalize here